src/gui/dataList.cpp

```diff
@@ -997,6 +997,37 @@ void FurnaceGUI::drawSampleList(bool asChild) {
   }
 }
 
+template <typename func_waveItemData>
```

Comment:
No new line needed after template<>

Reply:
👍 resolved this, it's also a pretty questionable use of template, it could also be a `struct WaveItemInfoGetter { virtual void GetInfo(...) = 0; };`, or take a vector of the necessary info -- lmk if you prefer another option like that